.shippable.yml

```diff
@@ -34,6 +34,16 @@ build:
         if [ "$IS_PULL_REQUEST" = "true" ]; then
           git rebase origin/${PULL_REQUEST_BASE_BRANCH};
         fi
+      - >
+         export NRF_HW_MODELS_VERSION=`cat boards/posix/nrf52_bsim/hw_models_version`;
```

Comment:
better define this under global section

Reply:
I don't think we can have this guy defined before the rebase just above.
But I moved it into its own line to make it more visible.